Examples/Framework/src/Framework/Sequencer.cpp

```diff
@@ -83,6 +84,11 @@ void ActsExamples::Sequencer::addWriter(std::shared_ptr<IWriter> writer) {
   ACTS_INFO("Added writer '" << m_writers.back()->name() << "'");
 }
 
+void ActsExamples::Sequencer::addAlias(const std::string& from,
+                                       const std::string& to) {
+  m_aliases[to] = from;
```

Comment:
This allows "overwriting" von aliasses. This is fine in principle, but it might lead to unexpected behaviour. From the function signature it could be assumed, that it is possible to have multiple aliases to a specific whiteboard entry, but that is not the case (because of the unique keys in the map).
Baybe it would be better to use something like 

```suggestion
  auto [it, success] = m_aliases.insert({to, from});
  if( not success ) {
    throw std::runtime_error("Alias to '" + to + "' already set");
  }
```

Reply:
Here `from` is the alias name and `to` is the name of the object it refers to, right? Would `aliasName` and `objectName` be clearer argument names?

I'm a bit confused about the `map[to]=from` (rather than how I'd naively expect `map[from]=to`, but I guess that's just how it is needed in the subsequent implementation (I didn't look at that yet).

If that's the case, couldn't we reasonably need multiple aliases pointing to the same object? Eg. different algorithms using the same input, but separately using aliases. I guess this bit could be handled with a `std::multimap`, but don't know if the rest of the implementation could cope with this situation.

Reply:
that is a good point. for now I would limit it explicitly that an object name can only be aliased once. whenever we see a need for enhancing this it should be easy to enable this, for example with a multimap

Reply:
I would also go with unique solution right now, to keep complexity as low as possible.